activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -804,6 +806,10 @@ def index_name_exists?(table_name, index_name, default)
       #
       #   add_reference(:products, :supplier, foreign_key: {to_table: :firms})
       #
+      # ====== Create an administrator_id column as a reference to the users table
+      #
+      #   add_reference(:products, :administrator, references: :users)
```

Comment:
This is achieved by above `add_reference(:products, :supplier, foreign_key: {to_table: :firms})`, not with `references: :table_name`